src/main/java/jenkins/plugins/logstash/persistence/BuildData.java

```diff
@@ -65,7 +65,8 @@
   public transient static final DateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
 
   public static class TestData {
-    int totalCount, skipCount, failCount;
+    int totalCount, skipCount, failCount, passCount;
+    List<Map<String, String>> failedTestsWithErrorDetail;
```

Comment:
I think Gson will support JavaBeans just fine, Can we have a simple DTO here instead of `Map<String,String>` , e.g.
```
class FailedTest {  final String fullName, errorDetails }
```